apps/login/src/app/(login)/idp/[provider]/success/page.tsx

```diff
@@ -139,6 +72,38 @@ export default async function Page(props: {
 
   const providerType = idpTypeToIdentityProviderType(idp.type);
 
+  if (link) {
+    if (!options?.isLinkingAllowed) {
+      return linkingFailed(branding, "Linking is no longer allowed");
```

Comment:
```suggestion
      // linking might have been disallowed since the invitation was created
      return linkingFailed(branding, "Linking is no longer allowed");
```